recipes/langtool-ignore-fonts

```diff
@@ -0,0 +1 @@
+(langtool-ignore-fonts :fetcher github :repo "cjl8zf/langtool-ignore-fonts" :files ("langtool-ignore-fonts.el"))
```

Comment:
If you use `:defaults` then you give your package room to grow into a multi-file package:
```suggestion
(langtool-ignore-fonts :fetcher github :repo "cjl8zf/langtool-ignore-fonts" :files (:defaults (:exclude "doc/*")))
```